pkg/compose/build.go

```diff
@@ -81,6 +81,14 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opti
 				Attrs: map[string]string{"ref": image},
 			})
 		}
+		if len(buildOptions.Platforms) > 1 {
+			buildOptions.Exports = []bclient.ExportEntry{{
+				Type: "image",
+				Attrs: map[string]string{
+					"push": "true",
+				},
+			}}
```

Comment:
Shouldn't this be `type: "docker"` + `"load": "true"` like the other one?

I believe this is legitimately trying to push to registry:
```shell
$ docker compose build hello
...
 => ERROR exporting to image                                                                                                                           0.7s
 => => exporting layers                                                                                                                                0.1s
 => => exporting manifest sha256:90bf7923cd97b6dec8f53161c7df1e6d2587ec52ccc060273cdc6b6ae8bd6c9c                                                      0.0s
 => => exporting config sha256:ec0a66e1fe3ca40d1042b63f3f82a0ff10189f86c6b68c1dd2b6f3213ac7057a                                                        0.0s
 => => exporting manifest sha256:8a1a9bea74c44f39ab83dd4a831d74845c1f4eda2c3f2a5c230e0988fa554236                                                      0.0s
 => => exporting config sha256:296e0f515945635ccc81e78c428bdf2b341a725f8ad033afbff0db2d0615aec1                                                        0.0s
 => => exporting manifest list sha256:87e79b4bda828912aba9e22edd32010ab65b56da406c307bf60b61e79616b8cb                                                 0.0s
 => => pushing layers                                                                                                                                  0.5s
------
 > exporting to image:
------
failed to solve: server message: insufficient_scope: authorization failed
```

Reply:
I just realized it's doing this because you can't export multi-platform images with the `docker` export type

I wonder if we need a `--output` flag on `build` with a default of `type=docker` that behaves the same as the buildx flag, so it can error out if you try to do `docker compose build --output=docker` on a multi-platform image.

Reply:
And TIL that `docker compose push` already exists 🤔 

But of course, it does not build:
```shell
$ docker compose push
An image does not exist locally with the tag: milas/hello
```

Reply:
Nope because `docker` exporter doesn't support multi-arch builds, I just need to remove the `push` option by default